src/Util/Strings.php

```diff
@@ -127,16 +132,15 @@ public static function getRandomName($len)
 
         $word = '';
 
-        for ($x = 0; $x < $len; $x ++) {
-            $r = mt_rand(0, count($table) - 1);
+        for ($x = 0; $x < $len; $x++) {
+            $r = random_int(0, count($table) - 1);
```

Comment:
Both Travis and Drone CI complains that `$r` is empty, so the next line `$word .= $table[$r]` fails because of an empty index.
You could set a breakpoint at https://github.com/friendica/friendica/blob/f300c8040fc1846866c95356fca5b39ef21843e3/tests/src/Util/StringsTest.php#L18 and step through the creation.

tbh I have no idea why this simple replacement causes an empty index.

Reply:
After the revert, travis succeed (the drone check isn't currently useful...). So it really was the change of these functions ... strange ..